app/views/user/new.html.haml

```diff
@@ -41,7 +41,7 @@
       .col-xs-2
         = f.label :role
       .col-xs-10
-        = f.select :role, options_for_select(UserHelper::ROLES_OPTIONS, f.object.role), class: "col-xs-10 col-sm-5"
+        = f.select(:role, User.roles.reject { |k,v| v==3 }.keys.map {|role| [role.titleize,role]}, :include_blank => false, class: "col-xs-10 col-sm-5")
```

Comment:
Hardcoded ID/Position. Also logic to get list of user should be moved to helper method.